lib/apachetest.pm

```diff
@@ -42,7 +42,7 @@ sub setup_apache2 {
     }
 
     # Make sure the packages are installed
-    zypper_call("in @packages");
+    zypper_call("--no-gpg-checks in @packages");
```

Comment:
`--no-gpg-checks` is used for packages repo which are not from DVD or SCC. Did you add a external 3rd party repo?

Reply:
Yes, it is related with the new cases, the new cases will re-register the PackageHub and this will introduce refreshing of gpg key, so I add this option for a workaround to avoid any rejection of the new key.